numpy/core/_methods.py

```diff
@@ -189,8 +189,25 @@ def _var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False):
     # Note that x may not be inexact and that we need it to be an array,
     # not a scalar.
     x = asanyarray(arr - arrmean)
+
+    # Complex types to -> (2,)float view for fast-path
+    _complex_to_float = {
```

Comment:
May as well make this global, no need to construct it every time.

Reply:
I had considered that as well - the overhead from building the dictionary seemed negligible so I had left it in, but I can just as well move it outside the function :+1:

Reply:
Let's move it out, if nothing else if makes `var` shorter and easier to read.